src/kakarot/instructions/system_operations.cairo

```diff
@@ -771,31 +774,46 @@ namespace CreateHelper {
             self=ctx.memory, element_len=size.low, element=bytecode, offset=offset.low
         );
 
-        // Get new account address
+        // Get new account address and increment nonce of deployer account
         let (state, evm_contract_address) = get_evm_address(
             ctx.state, ctx.call_context.address, popped_len, popped, size.low, bytecode
         );
         let (starknet_contract_address) = Account.compute_starknet_address(evm_contract_address);
-        tempvar address = new model.Address(starknet_contract_address, evm_contract_address);
+        tempvar target_address = new model.Address(starknet_contract_address, evm_contract_address);
```

Comment:
not sure about the renaming from `address` to `target_address`

Reply:
alright